<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>744</width>
    <height>531</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="styleSheet">
   <string notr="true"/>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="pushButton">
    <property name="geometry">
     <rect>
      <x>130</x>
      <y>350</y>
      <width>112</width>
      <height>34</height>
     </rect>
    </property>
    <property name="styleSheet">
     <string notr="true">color:green;

background-color: rgb(255, 255, 0);</string>
    </property>
    <property name="text">
     <string>Start</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_pic">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>370</y>
      <width>221</width>
      <height>81</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="Speedometer" name="widget" native="true">
    <property name="geometry">
     <rect>
      <x>40</x>
      <y>20</y>
      <width>311</width>
      <height>261</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_date">
    <property name="geometry">
     <rect>
      <x>150</x>
      <y>290</y>
      <width>171</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>10</pointsize>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="label_s">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>50</y>
      <width>81</width>
      <height>41</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <weight>75</weight>
      <bold>true</bold>
     </font>
    </property>
    <property name="text">
     <string>Station</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="lineEdit_s">
    <property name="geometry">
     <rect>
      <x>440</x>
      <y>60</y>
      <width>151</width>
      <height>25</height>
     </rect>
    </property>
   </widget>
   <widget class="QWidget" name="layoutWidget">
    <property name="geometry">
     <rect>
      <x>380</x>
      <y>100</y>
      <width>234</width>
      <height>21</height>
     </rect>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QLabel" name="label">
       <property name="text">
        <string>Train Number:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="label_train">
       <property name="font">
        <font>
         <weight>75</weight>
         <bold>true</bold>
        </font>
       </property>
       <property name="text">
        <string>Train Number:</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="Distance" name="widget_2" native="true">
    <property name="geometry">
     <rect>
      <x>500</x>
      <y>250</y>
      <width>211</width>
      <height>141</height>
     </rect>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>744</width>
     <height>31</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <widget class="QMenu" name="menuNew">
     <property name="title">
      <string>new</string>
     </property>
     <addaction name="separator"/>
     <addaction name="separator"/>
     <addaction name="separator"/>
    </widget>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="separator"/>
    <addaction name="menuNew"/>
   </widget>
   <widget class="QMenu" name="menuSetting">
    <property name="title">
     <string>Settings</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuOptions">
    <property name="title">
     <string>Options</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuView">
    <property name="title">
     <string>View</string>
    </property>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuSetting"/>
   <addaction name="menuOptions"/>
   <addaction name="menuView"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>Speedometer</class>
   <extends>QWidget</extends>
   <header>speedometer.h</header>
   <container>1</container>
   <slots>
    <signal>updateSpeed()</signal>
    <slot>QTimer()</slot>
   </slots>
  </customwidget>
  <customwidget>
   <class>Distance</class>
   <extends>QWidget</extends>
   <header location="global">distance.h</header>
   <container>1</container>
   <slots>
    <signal>updateDistance()</signal>
    <slot>QTimer()</slot>
   </slots>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>label_train</sender>
   <signal>customContextMenuRequested(QPoint)</signal>
   <receiver>pushButton</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>557</x>
     <y>166</y>
    </hint>
    <hint type="destinationlabel">
     <x>215</x>
     <y>416</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget</sender>
   <signal>updateSpeed()</signal>
   <receiver>widget</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>282</x>
     <y>325</y>
    </hint>
    <hint type="destinationlabel">
     <x>279</x>
     <y>263</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_2</sender>
   <signal>updateDistance()</signal>
   <receiver>widget_2</receiver>
   <slot>update()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>597</x>
     <y>409</y>
    </hint>
    <hint type="destinationlabel">
     <x>649</x>
     <y>425</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>widget_2</sender>
   <signal>updateDistance()</signal>
   <receiver>widget_2</receiver>
   <slot>QTimer()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>692</x>
     <y>424</y>
    </hint>
    <hint type="destinationlabel">
     <x>685</x>
     <y>376</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
